Upstream-Status: Backport
Signed-off-by: Ross Burton <ross.burton@arm.com>

From b1c71f19ea404e74d36ca5b8fbb0484043cdaef3 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@arm.com>
Date: Wed, 30 Jun 2021 11:25:01 +0100
Subject: [PATCH] tests: mark ul/ul as a known failure

As with ul/basic, this test produces different output when ran under
different terminals, which isn't very useful.

Set TS_KNOWN_FAIL so that these problems don't cause the test to fail.

Signed-off-by: Ross Burton <ross.burton@arm.com>
---
 tests/ts/ul/ul | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tests/ts/ul/ul b/tests/ts/ul/ul
index b856e33db..c707658ac 100755
--- a/tests/ts/ul/ul
+++ b/tests/ts/ul/ul
@@ -20,6 +20,9 @@ ts_init "$*"
 
 ts_check_test_command "$TS_CMD_UL"
 
+# This test provides different result on some terminals and virtual machines
+TS_KNOWN_FAIL="yes"
+
 printf "a\x08ab\x5F\x08c\\n\\ttab\\f\\b\\r" |
 	$TS_CMD_UL -t xterm >> $TS_OUTPUT 2>> $TS_ERRLOG
 
-- 
2.25.1

